Fix the accounts selector during intra-accounts funds transfer

dmigwi opened the pull request:
closes: https://github.com/crypto-power/cryptopower/issues/548

During Intra-Accounts funds transfer (transfer between accounts of the same wallet) only the mixed account are available for selection.

**To enable the intra-accounts transfer the following conditions are required:**
1. There is only one recipient instance available.
2. Both (i.e. source and recipient accounts) are from the same wallet.
3. Source account selected must have a spendable balance
4. Recipient's wallet tab option must be active.
5. The destination and source accounts must be different.

<h2>Source Account Selector When Intra-Accounts Selector is modified based on the above conditions</h2>
<img width="608" alt="Screenshot 2024-06-22 at 19 29 15" src="https://github.com/crypto-power/cryptopower/assets/22055953/9e7790b8-3e82-4f3c-a9a2-c0c3a02ef50d">

_NB: By default the info text above is always hidden until the info button is clicked._

<h2>Source Account Selector during all other funds transfer cases</h2>
<img width="702" alt="Screenshot 2024-06-22 at 19 28 53" src="https://github.com/crypto-power/cryptopower/assets/22055953/ef2aae1a-bd17-4e8d-9403-c1eab0bb9202">


_NB: By default the info text above is always hidden until the info button is clicked._

dreacot commented on ui/page/send/page.go:
```diff
@@ -218,7 +218,21 @@ func (pg *Page) initializeAccountSelectors() {
+				// 1. There is only one recipient instance available.
+				// 2. Both (i.e. source and recipient) use the same wallet.
+				// 3. Source account selected must have a spendable balance
+				// 4. Recipient's wallet tab option must be active.
```
this no.4 isn't clear what it means

dmigwi replied:
There are two tabs on the recipient info; "Address" and "Wallets". To activate the "wallets" tab, it needs to be currently on display.

dreacot commented:
i noticed you are following the pattern with the info icon present, but i believe that should be a modal instead, while the info icon should be right beside the from text

dmigwi commented:
> i noticed you are following the pattern with the present info icon, but i believe that should be a modal instead, while the info icon should be right beside the from text

@dreacot I am not sure I follow. I am lost because I have changed the current functionality of the info button. I just added more description text to the current functionality.
